### PR TITLE
Update docstrings to work better with Documenter.jl

### PR DESCRIPTION
**Pull request by abhro**
Please see commit messages for more details! Thank you!

### CODE REVIEW

File: src/sunpos.jl [hunk 1]
```diff
@@ -134,14 +146,4 @@ The returned `ra` and `dec` are in the given date's equinox.
...
-    ra = similar(jd, typej)
-    dec = similar(jd, typej)
-    longmed = similar(jd, typej)
-    oblt = similar(jd, typej)
-    for i in eachindex(jd)
-        ra[i], dec[i], longmed[i], oblt[i] = sunpos(jd[i], radians=radians)
-    end
-    return ra, dec, longmed, oblt
-end
+function sunpos end
```
Comment (giordano) [inline src/sunpos.jl:149]: Missing newline
```suggestion
function sunpos end

```

**Review by giordano — COMMENTED** [review]
Thanks!

File: src/sunpos.jl (hunk 1 above)
Comment (abhro) [reply src/sunpos.jl]: Done. Thanks!

File: src/trueanom.jl
Comment (abhro) [inline src/trueanom.jl:1]: Using unicode helps users who are getting help from the REPL

**Comment by abhro** [comment]
Hello! Are there any further fixes to make?

**Comment by abhro** [comment]
Thank you for merging!

**Comment by giordano** [comment]
Thank you for contributing!  I've invited you to the org, we're in need of more maintainers 🙂

**Pull request merged**